XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusManifestPreprocessor.cs

```diff
@@ -70,6 +70,27 @@ public static void GenerateManifestForSubmission()
 				Debug.LogWarning("Manifest error: unable to locate headset DoF mode");
 			}
 
+			int handTrackingTextIndex = manifestText.IndexOf("<!-- Request the headset handtracking mode -->");
+			if (handTrackingTextIndex != -1)
+			{
+
```

Comment:
I'm really confused about this class. It it part of the xrtk or does it belong to facebook?

Reply:
That is Oculus "stuff", trying to keep aligned to how they build their Manifest